markdown/en/ff-asn.md

```diff
@@ -42,7 +42,7 @@ by annotating it with the data-template attribute.
         <select data-container="hiddenLinks">
             <option>more ...</option>
             <!--
-                 This template is optional, if you dont supply an template it defaults to:
+                 This template is optional, if you dont supply one it defaults to:
```

Comment:
We are having two pieces of information here. This should really be two sentences.
We still need `don't` instead of `dont` and a comma after `one` as the sentence starts with `If`.
-> `This template is optional. If you don't supply one, it defaults to:`